models/api.topics.model.js

```diff
@@ -0,0 +1,10 @@
+const db = require("../db/connection");
+
+exports.fetchTopics = () => {
+  let sqlQuery = `SELECT * FROM topics `;
+  const queryValues = [];
+
```

Comment:
If this queryValues is not being used, is there a reason, or need to include it in your code?